### ISSUE TITLE
Title: Nutritive Analysis of Daily Foods [EDA] 

### ISSUE THREAD

**Issue by Varunshiyam** [Kushal997-das/Project-Guidance#1480]
### Initiative (Required)

GSSoC 2024 Extd 🚀

## 🗒️**Description**

This explores the Nutritional composition of everyday foods through exploratory data analysis(EDA). Using Python libraries such as pandas, NumPy, Matplotlib, Seaborn, we aim to uncover insights into the Nutritional landscape of common Diets.

------------

## 🎯**Goals**

- Uncover Nutritional Trends and Patterns 📊:

Explore nutrient distribution across different food groups and categories.
Identify correlations between various nutrients.

------------

- Gain Insights into Dietary Choices ✅ :

Analyze how frequently consumed foods contribute to overall nutrient intake.
Identify potential areas for improvement in dietary habits.

------------

- Develop Data Analysis and Visualization Skills :

Apply Python libraries effectively to clean, manipulate, and analyze nutritional data.
Create informative visualizations to communicate findings.

-----------
![Nutritive_Analysis](https://github.com/user-attachments/assets/6bec46e3-6f1e-455d-a0ab-30d9d3c2e04a)

## **Methodology**

**Data Acquisition and Preprocessing:**
- Download the Kaggle dataset using Python libraries (e.g., kaggle API).
- Perform data cleaning tasks: handle missing values, outliers, and inconsistencies.
- Transform data into a suitable format for analysis (e.g., create categorical variables, calculate ratios).
------------
**Exploratory Data Analysis:**
- Utilize pandas and NumPy for calculating descriptive statistics across food groups.
- Employ matplotlib and seaborn for creating histograms, boxplots, scatter plots, etc., - to reveal trends.
- Investigate nutrient correlations using techniques like Pearson's correlation coefficient.
------------
**Dietary Insights:**
- Analyze nutrient contributions from frequently consumed foods.
- Recommend improvements in dietary habits based on analysis.

------------

**Mentor:** 
- @SakiraAli1115 
-------------

**Kindly Assign this to me under Labels :**
☑️ 🌼 GSSoC 2024 Extd
☑️ 🍂 Hacktoberfest 2024

### Add any other context or screenshots about the feature request here.

<img width="1060" alt="Screenshot 2024-10-15 at 3 13 34 PM" src="https://github.com/user-attachments/assets/cfdcede6-55d2-4a35-98c4-a606316e240f">
<img width="935" alt="Screenshot 2024-10-15 at 3 13 56 PM" src="https://github.com/user-attachments/assets/3549a7a5-a174-4f90-9ec1-e1bac8ceb594">

**Comment by SakiraAli1115** [comment]
I am the mentor of this project.
Hello @Varunshiyam 
I appreciate your pull issue and thank you for your contribute!!!
@Kushal997-das kindly assign this issue!!!

**Comment by Varunshiyam** [comment]
This is a Remainder, Kindly Assign the Issue to me @Kushal997-das @sanjay-kv

**Comment by Kushal997-das** [comment]
@Varunshiyam, hello! I found that a similar issue was created and merged in another repository, which violates our rules. Therefore, I have marked this issue as spam, and you are no longer allowed to submit any PRs or issues in this repository.

For your reference, I can mention the PR, but I prefer not to, as it may affect your score. I hope you understand the importance of adhering to our open-source contribution guidelines.

Thank you!
